Ansible/inventories/inventory.kenet

```diff
@@ -0,0 +1,93 @@
+# This is a template for an inventory which can be used to configure a
```

Comment:
It would be better to keep the inventory clean - your group is ok, and the UI has been added to the group - but you will be picking up a lot of other variables from the other groups. So, clean out the inventory and keep only the things related to your site and your service (the UI)